solr/core/src/test/org/apache/solr/cloud/api/collections/SimpleCollectionCreateDeleteTest.java

```diff
@@ -141,7 +141,7 @@ public void testDeleteDoesNotDeleteSharedAutocreatedConfigSet() throws Exception
 
             // config for this collection is '.AUTOCREATED', and exists globally
             assertTrue(configName.endsWith(".AUTOCREATED"));
-            assertTrue(cloudClient.getZkStateReader().getConfigManager().listConfigs().contains(configName));
```

Comment:
There are a number of calls like this in tests.  The changed code is now kind of worse but I understand why it couldn't stay the same.  Perhaps SolrZkClient should have a listConfigs() ?